packages/legend-query-builder/src/components/fetch-structure/QueryBuilderOlapGroupByPanel.tsx

```diff
@@ -273,23 +282,6 @@ const QueryBuilderOlapColumnModalEditor = observer(
             }
           />
           <div className="query-builder__olap__modal__body">
-            <PanelFormSection>
-              <div className="panel__content__form__section__header__label">
-                OLAP Column Name
```

Comment:
Let's make sure we are consistent about the casing of `OLAP`, they should all CAPS I think (even for file name)